pkg/extensions/search/common/common_test.go

```diff
@@ -943,6 +943,20 @@ func TestBaseOciLayoutUtils(t *testing.T) {
 		So(size, ShouldBeZeroValue)
 	})
 
+	Convey("GetImageManifests", t, func() {
```

Comment:
In this case the test name is not representative of the actual behavior your are expecting and under what conditions.

Reply:
Indeed, removed